### PR TITLE
D1 change location date/time temp

### LINKED ISSUE #9: SIMULATOR DASHBOARD: change location, outside temperature, date & time
- On/Off button
 - Link to change location
 - Link to change outside temperature
 - Link to change date & time

### LINKED ISSUE #43: D3_Refactoring TimerTask
Refactoring the timerTask in the Dashboard controller
- help tracking the time to set the light on /off during away mode
- update house layout 
- support both timer or count down timer

### PR DESCRIPTION
**Pull request by kimchhengheng**
Fix time picker in simulator parameter
Fix time picker in dashboard
display the current logged user and role
display the time and date depend on current user

### CODE REVIEW

File: src/main/java/controllers/HouseLayoutController.java [hunk 1]
```diff
@@ -20,31 +20,35 @@
     private Main mainController;
 
     /**
-     * help to keep an instance of Main so we can call closeWindow method to close the current screen and load another screen
-     * @param mainController
```
Comment (huangs08) [inline src/main/java/controllers/HouseLayoutController.java]: I think your explanation is better than mine Kim, you can keep yours.

File: src/main/java/controllers/HouseLayoutController.java [hunk 2]
```diff
@@ -61,28 +65,30 @@ public String readFromFile(String url){
...
     }
+
     /**
-     * help to extract all the object by converting the json object to RoomModel object
-     * @param jsonText
-     * @return
+     * This method will extract the information from the json and place it in the room model array.
+     * @param jsonText string to be read
+     * @return array from room model that contains all the rooms in the house layout file.
+>>>>>>> 761a4d6e1e469fffeaaaa709802c6f8b86cde5fe
```
Comment (foreversponge) [inline src/main/java/controllers/HouseLayoutController.java]: This is from a merge conflict! I think it needs to be deleted.

Comment (kimchhengheng) [reply src/main/java/controllers/HouseLayoutController.java]: ok, i delete already

**Review by d-rossi — APPROVED** [review]
Looks good!

**Review by foreversponge — APPROVED** [review]

**Pull request merged**